tests/build.snapshot.js

```diff
@@ -1,8 +1,10 @@
+const byteDifferenceThreshold = 50;
+
 export const normalize = obj => {
 	let keys = Object.keys(obj);
 
-	if (keys.length === 1 && keys[0] === 'size' && typeof keys[0] === 'number') {
-		return { size: Math.round(obj.size / 10) * 10 };
+	if (keys.length === 1 && keys[0] === 'size' && typeof obj.size === 'number') {
```

Comment:
Turns out `typeof key === 'number'` is never true... wonder why...

Anyway sizes are now compared correctly and without it would never know that push-manifest.json included source maps. 🙈